dbt/include/fabric/macros/adapters/metadata.sql

```diff
@@ -9,6 +9,18 @@
   information_schema
 {%- endmacro %}
 
+{% macro get_use_database_sql(database) %}
+    {{ return(adapter.dispatch('get_use_database_sql', 'dbt')(database)) }}
+{% endmacro %}
+
+{%- macro default__get_use_database_sql(database) -%}
```

Comment:
Two comments:
- I think it makes more sense to define this as `synapse__get_use_database_sql` within `dbt-synapse`, rather than as `default__` implementation here
- You should explicitly `{{ return('') }}` - return an empty string - rather than returning it implicitly (by having an empty template)